src/fgdpp/fgdpp.cpp

```diff
@@ -268,45 +272,70 @@ FGD::FGD(std::string_view path, bool parseIncludes) {
 		exclusionList.emplace_back(path.data());
 
 		std::string_view dirPath = path.substr(0, path.find_last_of('/'));
-		std::smatch match;
-		std::regex exr{"@include+ \"(.*)\""};
 
-		while (std::regex_search(this->rawFGDFile, match, exr)) {
-			std::regex thisInclude("@include+ \"" + match[1].str() + "\"");
+        for(auto itr = rawFGDFile.begin(); itr != rawFGDFile.end(); itr++)
```

Comment:
Is this genuinely faster than regex? If it is, that's really sad and you can ignore this comment